package.json

```diff
@@ -28,7 +28,7 @@
     "copy:ext": "node ./tools/copy-ext.js",
     "format": "prettier --write '**'",
     "format:check": "prettier --check '**'",
-    "lint": "eslint src ./*.js",
+    "lint": "./tools/generate-all-locale.js && eslint src",
```

Comment:
Why the generate as part of linting?

Reply:
Without the file generated it will complain about the import here 
https://github.com/qlik-oss/sn-mekko-chart/blob/master/src/index.js#L31

![image](https://github.com/qlik-oss/sn-mekko-chart/assets/25658598/e6878348-15b1-4dfb-af06-757f5e12715e)

Reply:
You mean when linting so done on CI? Or locally?

Reply:
Both. The file all.js that is created by that script is not available until you run this (also not committed). I guess other repositories have a different setup for that

Reply:
Yes, I think ideally the `sn-gh-workflows` should be sure that the `all.json` file have been generated before runing a build and/ making a release.

Locally is another story. But I'd say since it already in the `build` script it won't really be a problem here.

Lets go with this for now and sort out a better common solution in different timeline 🙏

Reply:
Weirdly I do get a warning in vscode for the sn-table as well. However running lint locally seems to work 🤷

Reply:
Types:check fails on sn-table though